.travis.yml

```diff
@@ -3,8 +3,7 @@ sudo: required
 language: python
 services:
   - docker
-before_install:
-  - git clone https://github.com/pulp/pulp_file.git
+before_install: .travis/before_install.sh
```

Comment:
Please put a comment here on how you obtained the .sh & .py files from plugin-template, so that we can do it again in the future.